src/kOS/Suffixed/Part/PartValue.cs

```diff
@@ -61,7 +64,7 @@ private PartModuleFields GetModule(string modName)
             foreach (PartModule mod in Part.Modules)
             {
                 SafeHouse.Logger.Log(string.Format("Does \"{0}\" == \"{1}\"?", mod.moduleName.ToUpper(), modName.ToUpper()));
-                if (String.Equals(mod.moduleName, modName, StringComparison.CurrentCultureIgnoreCase))
+                if (String.Equals(mod.moduleName, modName, StringComparison.OrdinalIgnoreCase))
```

Comment:
Since we don't use have anything that's culture dependent,OrdinalIgnoreCase will be faster for the comparisons.  I've made an effort to make the change whenever I come across an instance, but we can certainly do a full refactor to catch all of them.